ynput/OpenPype
# Blender: Incorporate blender "Collections" into Publish/Load

## Changelog Description
Allow `blendScene` family to include collections.

## Additional info
The `blendScene` family now can include collections. The asset group for this family is now a collection, as Objects cannot hold collections. If parts of the objects loaded with the asset is linked in a local collection, it is preserved when updating the asset.

## Testing notes:
1. Try publish a `blendScene` asset that includes collections.
2. Try loading it, and check that nothing is missing.
3. Try adding objects and collections to different local collections.
4. Update the asset and check that those assets are still in the local collections.
